src/generics.rs

```diff
@@ -772,6 +772,27 @@ pub(crate) mod parsing {
 
             let begin = input.fork();
 
+            if cfg!(feature = "full") && input.peek(Token![use]) {
```

Comment:
I've gated this behind `cfg(feature = "full")` since that's what `~const` does below.